tests/test_modeling_common.py

```diff
@@ -3262,3 +3254,26 @@ def test_push_to_hub_dynamic_model(self):
         config = AutoConfig.from_pretrained(f"{USER}/test-dynamic-model", trust_remote_code=True)
         new_model = AutoModel.from_config(config, trust_remote_code=True)
         self.assertEqual(new_model.__class__.__name__, "CustomModel")
+
+
+@require_torch_gpu
+class LowCpuMemUsageTest(unittest.TestCase):
```

Comment:
No need to create a new class, you can add this test in the previous tester (not the one doing pushes to the Hub, the one before).